src/status_im/multiaccounts/core.cljs

```diff
@@ -43,13 +43,18 @@
 ;; available
 (defn contact-two-names
   "Returns vector of two names in next order nickname, ens name, display-name, three word name, public key"
-  [{:keys [names public-key] :as contact} public-key?]
+  [{:keys [names
+           compressed-key
+           public-key]
+    :as   contact} public-key?]
   (let [{:keys [nickname
                 ens-name
                 display-name
                 three-words-name]}
         (or names (contact-names contact))
-        short-public-key (when public-key? (utils/get-shortened-address public-key))]
+        short-public-key (when public-key?
+                           (or compressed-key
```

Comment:
compressed-key should be shortened as well, in form like `1234...5678`, its compressed but not short i believe

Reply:
It's ellipsed in the ui, I have checked (at least it looks like :) ), I think we use ellipse mode `middle`, we might want to have a custom method so the most relevant part is shown, but if that's ok, I'd do it in a separate PR

Reply:
yeah but ellipsed didn't work well, and in some places we don't have it, so we made this util function, btw i believe it should just work , you don't need to make any changes , `(utils/get-shortened-address (or compressed-key public-key))`